### PR TITLE
TL Translations for 1-git-internals.asc(402 words) and refspec.asc(718 words) - 1120 words 

### PR DESCRIPTION
**Pull request by teeheecakesph**
I have translated 1,120 words for this pull request. Please review my work. Thank you.

### CODE REVIEW

File: book/10-git-internals/1-git-internals.asc [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
-We found that understanding this information was fundamentally important to appreciating how useful and powerful Git is, but others have argued to us that it can be confusing and unnecessarily complex for beginners.
-Thus, we've made this discussion the last chapter in the book so you could read it early or later in your learning process.
-We leave it up to you to decide.
+Maari mong nalaktawan ang kabanatang ito mula sa mas naunang kabanata, o maari kang nakarating dito pagkatapos mo basahin ng sunod sunod ang aklat hanggang sa puntong ito -- sa ano mang kaso, dito tatalakayin ang mga panloob na pagtatrabaho at implementasyon ng Git. Nalaman namin na ang pag-intindi ng impormasyong ito ay mahalaga sa panimula upang mapahalagahan kung gaano kapaki-pakinabang at makapangyarihan ang Git, ngunit ang iba ay nakipagtalo sa amin na ito ay maaaring nakalilito at hindi kinakailangan kumplikado para sa mga nagsisimula. Kaya, ginawa namin ang dikusyong ito bilang huling kabanata sa aklat na ito upang mabasa mo ito nang maaga o huli sa proseso ng iyong pag-aaral. Iniwan na namin sa inyo ang pagpasya.
```
Comment (mind-blast) [inline book/10-git-internals/1-git-internals.asc]: Maari to Maaari

File: book/10-git-internals/1-git-internals.asc [hunk 2]
```diff
@@ -1,19 +1,13 @@
...
-In the last few years, the UI has been refined until it's as clean and easy to use as any system out there; however, the stereotype lingers about the early Git UI that was complex and difficult to learn.
+Sa mga naunang araw ng Git (halos bago ang 1.5), ang interface ng gumagamit ay mas kumplikado dahil binigyang diin nito ang filesystem sa halip na pakinisin ang VCS. Sa mga nakaraang taon, ang UI ay mas pinainam hanggang sa ito ay maging malinis at madaling gamitin tulad ng anumang mga sistema; subalit, ang gasgas tungkol sa naunang Git UI na kumplikado at mahirap pag-aralan ay nanatili.
 
-The content-addressable filesystem layer is amazingly cool, so we'll cover that first in this chapter; then, you'll learn about the transport mechanisms and the repository maintenance tasks that you may eventually have to deal with.
+Ang suson ng filesystem na maaring i-address ang nilalaman ay nakamamangha, kung kaya't tatalakayin natin yan ng una sa kabanatang ito; saka, matututuhan mo ang tungkol sa mga mekanismo ng transportasyon at ang mga gawain sa pagpapanatili ng imbakan na maari mong tingnan.
```
Comment (mind-blast) [inline book/10-git-internals/1-git-internals.asc]: Ang suson ng filesystem to Ang nilalaman-tinumbok na filesystem

Comment (mind-blast) [reply book/10-git-internals/1-git-internals.asc]: maari  to maaari

File: book/10-git-internals/1-git-internals.asc [hunk 3]
```diff
@@ -31,11 +25,8 @@ include::sections/maintenance.asc[]
...
 
-=== Summary
+=== Buod
 
-At this point, you should have a pretty good understanding of what Git does in the background and, to some degree, how it's implemented.
-This chapter has covered a number of plumbing commands -- commands that are lower level and simpler than the porcelain commands you've learned about in the rest of the book.
-Understanding how Git works at a lower level should make it easier to understand why it's doing what it's doing and also to write your own tools and helper scripts to make your specific workflow work for you.
+Sa puntong ito, dapat ay may magandang pag-unawa ka na kung ano ang ginagawa ng Git sa background at, sa ilang antas, kung paano ito ipinatupad. Ang kabanatang ito ay sumasakop ng ilang bilang ng mga utos sa plumbing -- ang mga utos na mas mababa at mas simple kumpara sa mga porselanang mga utos na iyong natutunan sa kabuuan ng aklat. Sa pamamagitan ng pag-intindi kung paana gumana ang Git sa mababang antas dapat mapadali nito ang pag-intindi kung bakit nito ginagawa ang ginagawa nito at ang pagsulat mo rin ng iyong sariling mga kasangkapan at mga script na tumutulong upang paganahin ang iyong partikular na workflow.
```
Comment (mind-blast) [inline book/10-git-internals/1-git-internals.asc]: plumbing to pag-plumb

File: book/10-git-internals/1-git-internals.asc [hunk 4]
```diff
@@ -31,11 +25,8 @@ include::sections/maintenance.asc[]
...
 
-Git as a content-addressable filesystem is a very powerful tool that you can easily use as more than just a VCS.
-We hope you can use your newfound knowledge of Git internals to implement your own cool application of this technology and feel more comfortable using Git in more advanced ways.
+Ang Git bilang isang filesystem na maaring i-address ang nilalaman ay isang makapangyarihang kasangkapan na madaling gamitin higit pa sa VCS. Umaasa kami na maari mong gamitin ang iyang bagong kaalaman sa panloob ng Git upang maipatupad ang iyong sariling nakakamanghang application sa teknolohiyang ito at maging mas komportable sa paggamit ng Git sa mas mataas na antas na paraan.
```
Comment (mind-blast) [inline book/10-git-internals/1-git-internals.asc]: maaring  to maaaring

Comment (mind-blast) [reply book/10-git-internals/1-git-internals.asc]: maari  to maaari

File: book/10-git-internals/sections/refspec.asc [hunk 5]
```diff
@@ -1,8 +1,8 @@
 [[_refspec]]
 === Ang Refspec
 
-Throughout this book, we've used simple mappings from remote branches to local references, but they can be more complex.
-Suppose you were following along with the last couple sections and had created a small local Git repository, and now wanted to add a _remote_ to it:
+Sa buong aklat na ito, ginamit namin ang simpleng pagmapa mula sa remote na mga sangay patungo sa mga lokal na reperensiya, ngunit maaring maging mas komplikado ang mga ito.
```
Comment (mind-blast) [inline book/10-git-internals/sections/refspec.asc]: no need to change branch .. please change back sangay to branch

Comment (mind-blast) [reply book/10-git-internals/sections/refspec.asc]: maaring  to maaaring

File: book/10-git-internals/sections/refspec.asc [hunk 6]
```diff
@@ -9,7 +9,7 @@ Ipagpalagay na sumunod ka sa mga huling pares na mga seksyon at nakagawa ng isan
 $ git remote add origin https://github.com/schacon/simplegit-progit
 ----
 
-Running the command above adds a section to your repository's `.git/config` file, specifying the name of the remote (`origin`), the URL of the remote repository, and the _refspec_ to be used for fetching:
+Ang pagpapatakbo ng command na nasa ibabaw ay magdaragdag ng seksyon sa `.git/config` na file ng iyong repository, na tumutukoy sa pangalan ng remote (`origin`), ang URL ng remote na repository, at ang _refspec_ na gagamitin para sa pagkuha:
```
Comment (mind-blast) [inline book/10-git-internals/sections/refspec.asc]: command  to utos
repository to repositoryo

File: book/10-git-internals/sections/refspec.asc [hunk 7]
```diff
@@ -21,8 +21,8 @@ Ang pagpapatakbo ng command na nasa ibabaw ay magdaragdag ng seksyon sa `.git/co
 Ang format ng refspec ay, una, ang opsyonal na `+`, sinusundan ng `<src>:<dst>`, kung saan ang `<src>` ay ang pattern para sa mga reperensiya sa remote na bahagi at `<dst>` ay kung saan ang mga reperensiya ay susubaybayan nang lokal.
 Ang `+` ay nagsasabi sa Git na i-update ang reperensiya kahit na ito ay hindi isang mabilis na pasulong.
 
-In the default case that is automatically written by a `git remote add` command, Git fetches all the references under `refs/heads/` on the server and writes them to `refs/remotes/origin/` locally.
-So, if there is a `master` branch on the server, you can access the log of that branch locally via any of the following:
+Sa default na kaso na awtomatikong isinulat ng isang `git remote add` na command, kinukuha ng Git ang lahat ng mga reperensiya sa ilalim ng mga `refs/heads/` sa server at sinusulat ito sa `refs/remotes/origin/` nang lokal.
```
Comment (mind-blast) [inline book/10-git-internals/sections/refspec.asc]: command to utos

File: book/10-git-internals/sections/refspec.asc [hunk 8]
```diff
@@ -21,8 +21,8 @@ Ang pagpapatakbo ng command na nasa ibabaw ay magdaragdag ng seksyon sa `.git/co
...
+Kaya, kung mayroong isang master na sanga sa server, maaari mong ma-access ang log ng sanga na iyon nang lokal gamit ang alinman sa mga sumusunod:
```
Comment (mind-blast) [inline book/10-git-internals/sections/refspec.asc]: sanga  to branch

File: book/10-git-internals/sections/refspec.asc [hunk 9]
```diff
@@ -31,9 +31,9 @@ $ git log remotes/origin/master
 $ git log refs/remotes/origin/master
 ----
 
-They're all equivalent, because Git expands each of them to `refs/remotes/origin/master`.
+Lahat ng mga ito ay magkatumbas, sapagkat pinapalawak ng Git ang mga ito sa `refs/remotes/origin/master`.
 
-If you want Git instead to pull down only the `master` branch each time, and not every other branch on the remote server, you can change the fetch line to refer to that branch only:
+Kung nais mong kunin ng Git ang `master` na sanga lang sa bawat pagkakataon, at hindi lahat ng ibang mga sanga sa remote na server, maari mong baguhin ang linya ng pagkuha upang sumangguni sa sanga na iyon lamang:
```
Comment (mind-blast) [inline book/10-git-internals/sections/refspec.asc]: sanga to branch

**Comment by mind-blast** [comment]
Please check on my suggestion. Thank you.

File: book/10-git-internals/1-git-internals.asc (hunk 1 above)
Comment (teeheecakesph) [reply book/10-git-internals/1-git-internals.asc]: Done. Changed to Maaari.

File: book/10-git-internals/1-git-internals.asc (hunk 2 above)
Comment (teeheecakesph) [reply book/10-git-internals/1-git-internals.asc]: Done. Changed the translation based on the comments

File: book/10-git-internals/1-git-internals.asc (hunk 3 above)
Comment (teeheecakesph) [reply book/10-git-internals/1-git-internals.asc]: Done.

File: book/10-git-internals/1-git-internals.asc (hunk 4 above)
Comment (teeheecakesph) [reply book/10-git-internals/1-git-internals.asc]: Done.

**Comment by teeheecakesph** [comment]
Done, updated the translations based on robin-ho's comments.

**Pull request merged**